pkg/utils/api_conversion/trace.go

```diff
@@ -117,6 +118,8 @@ func isResourceGateway(resource *gloov1.SourceMetadata_SourceRef) bool {
 func gatewayKindsMap() map[string]bool {
 	return map[string]bool{
 		resources.Kind(new(gatewayv1.Gateway)): true,
+		// https://github.com/solo-io/gloo/blob/main/projects/gateway2/translator/listener/gateway_listener_translator.go#L492
```

Comment:
[nit] permalink?

Reply:
Does this just get rendered as `md`>

Reply:
Does it get rendered out? I thought it was just a comment for https://github.com/solo-io/gloo/blob/87603001fdb2e37377039fa897df2569e1d694c2/projects/gateway2/translator/listener/gateway_listener_translator.go#L492